menu/menu.yaml

```diff
@@ -386,6 +386,8 @@
       Articles:
       - Url: nservicebus/hosting/azure-functions/service-bus
         Title: Service Bus
+      - Url: nservicebus/hosting/azure-functions/custom-triggers
```

Comment:
I didn't add it to the menu intentionally. If we want to add it, it should be a sub-item of "Service Bus", not on the same level.

Reply:
Problem is, that level of nestedness is not supported. How about naming it "Service Bus custom triggers"?